docs/install/index.md

```diff
@@ -1125,7 +1125,8 @@ You can [build MXNet-R from source](windows_setup.html#install-mxnet-package-for
   options(repos = cran)
   install.packages("mxnet")
 ```
-Change cu92 to cu80, cu90 or cu91 based on your CUDA toolkit version. Currently, MXNet supports these versions of CUDA.
+Change cu92 to cu90, cu91 or cuda100 based on your CUDA toolkit version. Currently, MXNet supports these versions of CUDA.
```

Comment:
I think it would be more clear to say that MXNet supports these versions of CUDA **on Windows**. Doesn't 8.0 still work with Linux? Also, if this is some kind of bug, I'd footnote it, referencing the issue. Maybe somebody out there really wants 8 and can get it work.

Reply:
We don't publish Linux packages for R. 
It's only OSX and Windows.